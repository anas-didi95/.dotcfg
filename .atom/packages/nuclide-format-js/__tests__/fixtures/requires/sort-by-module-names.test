const p = require('C');
const q = require('A');
const r = require('b');
const U = require('b');
const V = require('C');
const W = require('A');
import type X from 'b';
import type Y from 'C';
import type Z from 'A';
require('A');
require('b');
require('C');

const m: [X, Y, Z] = p(q(r(U, V, W)));
