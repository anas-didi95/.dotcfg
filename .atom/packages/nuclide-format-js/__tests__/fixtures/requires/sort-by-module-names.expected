import type Z from 'A';
import type X from 'b';
import type Y from 'C';

require('A');
require('b');
require('C');

const W = require('A');
const U = require('b');
const V = require('C');

const q = require('A');
const r = require('b');
const p = require('C');

const m: [X, Y, Z] = p(q(r(U, V, W)));
